resourcemanager/resourcemanager.go

```diff
@@ -84,6 +84,33 @@ func (rm *ResourceManager) UpdateResources() {
 	}
 }
 
+func (rm *ResourceManager) RemovedInvalidUIDs() {
+	resources := rm.store.GetResources()
+
+	for _, r := range resources {
+		members := rm.store.GetMembers()
+		for _, m := range members {
+			if m.Level != uint8(models.Inactive) {
+				return
+			}
+
+			if len(m.RFID) == 0 {
+				return
+			}
+
+			/* We will just try to remove all invalid members even if they are already removed */
+			b, _ := json.Marshal(&models.AddMemberRequest{
+				ResourceAddress: r.Address,
+				Command:         "deletuid",
```

Comment:
If this isn't a typo then would it be alright if you could a const for these rfid commands? So that it's known and not a mistake?

Reply:
Soooo, would it be alright if we could create a const for these? So that the standard is known :upside_down_face: